pipeline/outputs/prometheus-remote-write.md

```diff
@@ -13,6 +13,9 @@ Important Note: The prometheus exporter only works with metric plugins, such as
 | host                 | IP address or hostname of the target HTTP Server                                                                                                                                                                                                                       | 127.0.0.1 |
 | http_user            | Basic Auth Username                                                                                                                                                                                                                                                    |           |
 | http_passwd          | Basic Auth Password. Requires HTTP_user to be set                                                                                                                                                                                                                      |           |
+| AWS\_Auth                  | Enable AWS SigV4 authentication                                                                                                                                                                                                                                                                                                    | false     |
+| AWS\_Service               | Specify the AWS service code, i.e. es, xray, etc., of your service, used by SigV4 authentication. Usually can be found in the service endpoint's subdomains, `protocol://service-code.region-code.amazonaws.com`                                                                                                                   | aps          |
+| AWS\_Region                | Specify the AWS region of your service, used by SigV4 authentication                                                                                                                                                                                                                                                               |           |
```

Comment:
same as above, we can be specific => region of your Amazon Managed Service for Prometheus workspace.